### DISCUSSION TITLE
How to change collection API Authorization to user

### DISCUSSION THREAD

**Discussion by kaschula** [pocketbase/pocketbase#791] [Q&A] [answered]
Hello, I'm struggling to find in the docs how to change a collections API Authorization from the default which seems to be admin to user. 

I'm sure I'm missing something obvious.

Thanks

**Comment by ganigeorgiev** [comment, marked as answer]
You can change the default API rules from _"Collections > Edit collections (the settings cogwheel) > API rules tab"_.

More info on the collection api rules could be found in https://pocketbase.io/docs/manage-collections/#collection-api-rules.

**Reply by kaschula** [reply]
Thank you so much
